src/trace.c

```diff
@@ -62,12 +62,12 @@ static const char *rs_severities[] = {
  * perhaps, and an error dialog for a browser.
  *
  * \todo Do we really need such fine-grained control, or just yes/no tracing? */
-LIBRSYNC_EXPORT void rs_trace_to(rs_trace_fn_t *new_impl)
+void rs_trace_to(rs_trace_fn_t *new_impl)
```

Comment:
Please don't change anything inside the `src` directory. It will make any update from upstream a nightmare. This repo only adds a `static` directory with the preconfigured `config.h` for each supported platform.